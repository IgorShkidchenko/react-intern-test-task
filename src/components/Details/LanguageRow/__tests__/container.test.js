import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import LanguageRowContainer from "../container";

configure({ adapter: new Adapter() });

describe("<LanguageRowContainer />", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<LanguageRowContainer percentage="1" languageName="name" />);

    expect(wrapper).toMatchSnapshot();
  });
});
