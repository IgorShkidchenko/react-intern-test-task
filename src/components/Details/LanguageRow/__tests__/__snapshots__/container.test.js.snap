// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LanguageRowContainer /> renders correctly 1`] = `
<LanguageRowComponent
  languageName="name"
  percentage="1"
/>
`;
